app/views/layouts/admin_layout.html.erb

```diff
@@ -11,18 +11,6 @@
     navigation_items: navigation_items
   } %>
 
-  <div class="govuk-width-container">
-    <% if breadcrumbs.links.any? %>
-      <%= render "govuk_publishing_components/components/breadcrumbs", {
-        collapse_on_mobile: true,
-        breadcrumbs:
-          breadcrumbs
-            .links
-            .reject { |link| link.current? }
-            .map { |link| { title: link.text, url: link.url } }
-      } %>
-    <% end %>
```

Comment:
**issue:** scope

Transition is using two different layouts depending on the page: one is on the Design System (mostly for newer pages), while the other is using the Bootstrap gem (mostly for older, often more complex pages)

This file is the layout that uses the Design System, and the breadcrumbs are outside the `main` element here (the `main` element is just below), so I don't think we need to remove the breadcrumbs here. The other file `application.html.erb` is the one using the Bootstrap layout, so that's the only one from which we should need to remove breadcrumbs

Reply:
Cool, I'll correct that